cylc/flow/prerequisite.py

```diff
@@ -156,7 +165,7 @@ def is_satisfied(self):
             return self._all_satisfied
         else:
             # No cached value.
-            if not self.satisfied:
+            if self.satisfied == {}:
```

Comment:
The former way is more Pythonic, innit?

Reply:
I thought it was misleading at a casual glance
```python
def is_satisfied(self):
    if not self.satisfied:
        return True
```

Reply:
OK, I see your point, I didn't look at the surrounding context. Although the comment on the next line explains what's going on, at least. Probably indicates a badly named dict, if anything (my fault).